decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -403,20 +403,16 @@ def self.valuator_role_ids_has(value)
         where(query, value:)
       end
 
-      def self.ransackable_scopes(auth_object = nil)
-        base = [:with_any_origin, :with_any_state, :state_eq, :voted_by, :coauthored_by, :related_to, :with_any_taxonomies]
-        return base unless auth_object&.admin?
-
-        # Add extra scopes for admins for the admin panel searches
-        base + [:valuator_role_ids_has]
+      def self.ransackable_scopes(_auth_object = nil)
```

Comment:
"Process admins" such as moderators, collaborators, valuators or admin user roles have access to those lists and they are not _admins_ as in `current_user.admin?`. 

I was about to refactor this to check the user roles but then I thought that it would add a lot of complexity and it might not be necessary at all. At the end, this only gives you filtering and sorting capabilities to your scope.